### ISSUE TITLE
Improve the schemes preview HTML page

### ISSUE THREAD

**Issue by alexbooker** [base16-builder/base16-builder#96]
In commit 5ec4638349e46883781db351cec5a5db564464bf I added a simple `ls schemes` command:

![](http://i.imgur.com/8F1NhEp.png)

It has a lot of room for improvement :wink: 

In the [original issue](#33) @Evinon contributed a mock design I like:

![](https://camo.githubusercontent.com/67574aad4316912e33e0cd344cf66e26a17bee84/687474703a2f2f61626c6f61642e64652f696d672f6465736b746f706864327836797339352e706e67)

And @CrashyBang offered implement something similar using [Code Mirror](http://codemirror.net/):

![](https://cloud.githubusercontent.com/assets/2978850/14595847/7f851148-0596-11e6-83a5-ee25f69f11c7.png)

I'm opening this issue to keep track of the changes.

**Comment by CrashyBang** [comment]
Hey @alexbooker,

Do we have any documentation on getting set up?

Cheers.

**Comment by CrashyBang** [comment]
Hey @alexbooker,

Do we have any documentation on getting set up?

Cheers.

**Comment by alexbooker** [comment]
Does [this](https://github.com/base16-builder/base16-builder/blob/master/contributing.md#development) help?

**Comment by CrashyBang** [comment]
Hey @alexbooker,

Sweet just what I was looking for (sorry didn't scroll to the bottom when I checked contributing) was going to offer to write some up if needed.

Cheers.

**Comment by alexbooker** [comment]
No worries. I think `Contributing.md` is _way_ too convoluted at the moment. It has some really important information by @aloisdg but I want to streamline the most commonly-needed information.

**Comment by CrashyBang** [comment]
@alexbooker I think that would be a really good idea, also I wanted to ask after adding you on hangouts, is there any reason we are not using [Gitter](https://gitter.im) I think it would be a really handing for quick one off questions like the above and it's much nice for on going conversations like this?

**Comment by alexbooker** [comment]
You're right, we should be using Gitter. [I'm in the room now](https://gitter.im/base16-builder/base16-builder).

**Comment by alexbooker** [comment]
Thank you @ClaytonPassmore for your progress on this :+1:!
